ext/handlers/chatmember.go

```diff
@@ -0,0 +1,39 @@
+package handlers
+
+import (
+	"fmt"
+
+	"github.com/PaulSonOfLars/gotgbot/v2"
+	"github.com/PaulSonOfLars/gotgbot/v2/ext"
+	"github.com/PaulSonOfLars/gotgbot/v2/ext/handlers/filters"
+)
+
+type ChatMember struct {
+	Response Response
+	Filter   filters.ChatMember
+}
+
+func NewChatMember(f filters.ChatMember, r Response) ChatMember {
+	return ChatMember{
+		Response: r,
+		Filter:   f,
+	}
+}
+
+func (c ChatMember) CheckUpdate(b *gotgbot.Bot, u *gotgbot.Update) bool {
+	if u.ChatMember != nil {
+		if c.Filter != nil {
+			return c.Filter(u.ChatMember)
+		}
+		return true
+	}
+	return false
```

Comment:
```suggestion
	return u.ChatMember != nil && c.Filter != nil && c.Filter(u.ChatMember)
```
This can be simplified into a single line; reducing unnecessary if statements improves readability

Reply:
i did that cuz i considered what if a user provides nil for the filter in constructor.

Reply:
The filter check is fine, the issue is the unnecessary double if statement which can be simplified